docs/readmes/lte/sessiond.md

```diff
@@ -152,4 +152,18 @@ Final Unit Indication AVP is needed to trigger any action by SessionD.
 Note when SessionD receives a grant which includes Final Unit Indicaton, it 
 will continue tracking this charging key until any of its quotas is exhausted. 
 Once it is exhausted, Final Unit Action (termnate, redirect, restrict) 
-will be executed.
\ No newline at end of file
+will be executed.
+
+#### Enabling Restriction
+In addition to Final Unit Action AVP, OCS must include a list of Filter-ID AVP in the
+final grant. These filters reference static policy rule identifiers that are pre-provisioned
+in PCRF and should contain an explicit DENY rule for non allowed traffic.
+
+Note that static rules referenced in Filter-ID must not be activated by Gx Interface due to a
```

Comment:
What is this limitation in PipelineD? So we cannot have a same rule installed as both Gx and Gy apps?

Reply:
In the current implementation, we are using same stats ovs table for gy and enforcement rules.
This might trigger an issue when the same rule is active in both tables for the same user. (Pending more testing once the PR for pipelined is merged)

Reply:
Ok, if this is the case, then we should make sure to remove all gx rules before installing the restrict rules, correct?